hw1/02-palindrome.js

```diff
@@ -1,3 +1,23 @@
 const elem = document.querySelector('input');
-
+const resultText = document.getElementById('resultText');
 elem.addEventListener('input', handleInput);
+
+function handleInput(){
+    let input = parseInt(elem.value);
+    if(input >= 0){
+        let reverseInput = parseInt(String(elem.value).split('').reverse().join(''));
+        if( input == reverseInput){
+            resultText.style.color = 'green';
+            resultText.innerHTML = 'Yes. This is a palindrome!';
+        }else{
+            resultText.style.color = 'red';
+            resultText.innerHTML = 'No. Try again.'
```

Comment:
I think we might want to use these [colors](https://getbootstrap.com/docs/4.0/utilities/colors/) instead of changing the color directly

Reply:
Done!